components/CacheIllustration.tsx

```diff
@@ -0,0 +1,82 @@
+import Image from "next/image";
+
+export const CacheAnimation: React.FC<{
+  location?: string;
+  skipCache: boolean;
+}> = ({ location, skipCache }) => {
+  return (
+    <>
+      <div
+        className="accelerate-image-group"
+        style={{
+          width: 400,
+          padding: "10px",
+        }}
+      >
+        <div
+          style={{
+            fontSize: "12px",
+            textAlign: "left",
+            display: "relative",
+          }}
+        >
+          {location ? <span>{location}</span> : <br></br>}
+          <Image
+            width={20}
+            height={20}
+            className="accelerate-server-img"
+            src="/server.svg"
+            alt="Server icon"
+          ></Image>
+          <p>Vercel Edge Server</p>
```

Comment:
is it possible to have the arrows only move _while_ the speed test is being executed and stop them right afterwards?

Reply:
@nikolasburk You mean for these? 
<img width="1043" alt="Screenshot 2023-07-06 at 15 42 31" src="https://github.com/prisma/accelerate-speed-test/assets/14851246/8033f073-a1a1-4464-919a-b6916647b0c9">

Reply:
yes 👍 basically, there shouldn't be any movement when the test is not running (because at that time, there's actually no communication between the components). only when the speed test is running, data is flowing between the components, so semantically it feels more accurate when the arrows are only moving while the test is running.

Reply:
Done! 🙂